### DISCUSSION TITLE
Getting no close frame received or sent error

### DISCUSSION THREAD

**Discussion by devashish-gopalani-cognoai** [deepgram/community#498] [General help]
### Which Deepgram product are you using?

Deepgram SDKs

### Details

Currently I am using the deepgram-sdk==3.0.0rc1 version of the Deepgram SDK. 
Suddenly out of nowhere when I am calling the deepgram.send() function, I am getting the below error:

> {'type': 'Exception', 'description': 'Unknown error _listening', 'message': 'no close frame received or sent', 'variant': ''}

What could be the probable cause for this? How should I fix this?

The details of the model which I am using is - 
model = nova-general
language = hi-Latn
endpointing = 500
interim results = true
sample rate = 8000
encoding = linear16

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment]
@devashish-gopalani-cognoai  👋  if this related to our `Python SDK` trying [using our 3.0](https://github.com/deepgram/deepgram-python-sdk/releases/tag/v3.0.0) release which we just pushed out this week. Else, if it's another SDK just let us know which one.  cc @dvonthenen

**Comment by davidvonthenen** [comment]
This might be the same person... if not, it's the similar issue:
https://discord.com/channels/1108042150941294664/1185789147525414983

**Comment by devashish-gopalani-cognoai** [comment]
@jpvajda @dvonthenen I have upgraded to 3.0. Will update you if I face any issue again. Till then, I am closing this issue. Thanks for the support!
